android/capacitor/src/main/java/com/getcapacitor/MessageHandler.java

```diff
@@ -35,40 +35,36 @@ public void postMessage(String jsonStr) {
 
       String type = postData.getString("type");
 
-      if (type != null && type.equals("cordova")) {
-        String callbackId = postData.getString("callbackId");
+      boolean typeIsNotNull = type != null;
+      boolean isCordovaPlugin = typeIsNotNull && type.equals("cordova");
+      boolean isJavaScriptError = typeIsNotNull && type.equals("js.error");
+
+      String callbackId = postData.getString("callbackId");
+
+      if (isCordovaPlugin) {
         String service = postData.getString("service");
         String action = postData.getString("action");
         String actionArgs = postData.getString("actionArgs");
-        Logger.verbose(Logger.tags("Plugin"), "To native (Cordova plugin): callbackId: " + callbackId + ", service: " + service +
-          ", action: " + action + ", actionArgs: " + actionArgs);
+
+        Logger.verbose(Logger.tags("Plugin"), "To native (Cordova plugin): callbackId: " + callbackId + ", service: " + service + ", action: " + action + ", actionArgs: " + actionArgs);
+
         this.callCordovaPluginMethod(callbackId, service, action, actionArgs);
       } else if (type != null && type.equals("js.error")) {
```

Comment:
Did you mean to use the new `isJavaScriptError` bool here?

Reply:
Good spot, fixed!